src/exec.ts

```diff
@@ -109,7 +117,11 @@ export function isInPath(command: string) {
       }
     } else {
       const res = childProcess.spawnSync('whereis', [command]);
-
+      if (res.error && (res.error as SpawnSyncError).code === 'ENOENT') {
+        // If `whereis` doesn't exist we assume that the command exists in the path. It might
+        // not but it's better to try and fail than to not try at all.
+        return true;
```

Comment:
What if we just notify the user instead? :) 
```suggestion
       throw new Error(`Error trying to locate binary for "${command}". Make sure that 'whereis' is installed on your system.`);
```

Reply:
Sure, that works just a fine for me. :)